### PR TITLE
consoletools: copy utils to bin

### PR DESCRIPTION
**Pull request by ebzzry**

### CODE REVIEW

**Comment by jagajaga** [comment]
cc @edolstra

File: pkgs/os-specific/linux/consoletools/default.nix [hunk 1]
```diff
@@ -10,14 +10,26 @@ stdenv.mkDerivation rec {
   };
 
   buildInputs = [ SDL ];
+
   makeFlags = [ "DESTDIR=$(out)"];
 
+  installPhase = ''
+   mkdir -p $out/bin
+   cp utils/ffcfstress $out/bin
```
Comment (joachifm) [inline pkgs/os-specific/linux/consoletools/default.nix]: Equivalently, `cp -t $out/bin utils/*`

**Comment by ebzzry** [comment]
@joachifm fixed (∩_∩)

**Comment by joachifm** [comment]
I wonder, does overriding the entire `installPhase` affect the output at all? Are you sure that whatever it used to install into the output is included still?

**Comment by ebzzry** [comment]
Currently on master, the installing profile does not have access to the binaries because they’re not copied to the correct directory.

**Comment by joachifm** [comment]
@ebzzry what I mean is that now *only* the binaries are installed.

**Comment by joachifm** [comment]
Actually, I just looked a bit closer and my suggestion is all wrong, as it would copy the source files into bin as well.

The actual problem is that `make install` puts things into `$out/usr/local`; overriding the entire `installPhase` is no good here.

**Comment by joachifm** [comment]
`make install` uses `$(DESTDIR)$(PREFIX)`, so what you want to do is set `installFlags = ''PREFIX=""''`.

**Comment by ebzzry** [comment]
@joachifm fixed =)

**Comment by joachifm** [comment]
Thank you

**Comment by ebzzry** [comment]
@joachifm Thanks! (/^▽^)/

**Pull request merged**